synapse/push/httppusher.py

```diff
@@ -294,8 +294,13 @@ def dispatch_push(self, event, tweaks, badge):
             defer.returnValue([])
         try:
             resp = yield self.http_client.post_json_get_json(self.url, notification_dict)
+            logger.info(
+                "Sent to: %s push-notify: %r get response: %r" % (
+                    self.url, notification_dict, resp
+                )
+            )
         except:
-            logger.warn("Failed to push %s ", self.url)
+            logger.debug("Failed to push %s %r" % (self.url, notification_dict))
```

Comment:
Sorry, I meant having two log lines, as we still want the warning, e.g.:

```
logger.warn("Failed to push %s ", self.url)
logger.debug("Notification dict: %r", notification_dict)
```
Sorry, I meant having two log lines, as we still want the warning, e.g.:

```
logger.warn("Failed to push %s ", self.url)
logger.debug("Notification dict: %r", notification_dict)
```